apps/microtvm/zephyr/demo_runtime/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -21,7 +21,7 @@
 CONFIG_CMSIS_DSP=y
 
 # Required for Cortex-M33 devices.
-CONFIG_MAIN_STACK_SIZE=1536
```

Comment:
i don't think we should mess with these now--we should keep them as minimal as possible under _some_ runtime configuration. maybe we can create AOT versions which can be selected with an option? after we have project generator, it should be easy to customize these.

it also might be easiest, considering the other changes, to just create `apps/microtvm/aot_demo`.

Reply:
yeah I like this better. At first I tried to introduce less redundancy, but it makes it more complicate to understand
I renamed `demo_runtime` to `host_driven_demo` and created another demo as `aot_demo`.